examples/dataset/object_detection_2d/pyproject.toml

```diff
@@ -0,0 +1,17 @@
+[tool.poetry]
+name = "object-detection-2d"
+version = "0.1.0"
+description = ""
```

Comment:
plz check other similar files to make sure things are consistent.

Reply:
updated it to match others